### ISSUE TITLE
One avatar with invalid attribution format breaks entire avatar browser

### ISSUE THREAD

**Issue by wmurphyrd** [mozilla/hubs#4182]
**Description**
Sometimes, when creating an avatar, the "attribution" field ends up saved as plaintext instead of JSON. If that one avatar gets approved or featured, then no avatars will appear in the avatar browser. 

**To Reproduce**
We're not sure exactly how it got to be this way, but here's a screenshot of an offending avatar:

![badavatar](https://user-images.githubusercontent.com/10034859/115755525-0ec7d880-a363-11eb-83f2-7960d0d669e3.png)

When this avatar is featured, then the avatar browser looks like this:

![no avatars](https://user-images.githubusercontent.com/10034859/115755667-3454e200-a363-11eb-8c77-e53deb46415d.png)

And this network console shows 500 errors on the attempts to fetch avatar listings

**Expected behavior**

Still see avatars in the listing, validate the attributions field on import/update

**Hardware** 
 - Device: Desktop
 - OS: Linux
 - Browser: Firefox

**Comment by Barbe972** [comment]
Hello 
Do you have one solution for this problem I have the same things on Hubs Cloud machine
I can't see avatars on my project, all the other functionality are running well I can't choose an avatar

![hubs avatars](https://user-images.githubusercontent.com/90916859/136253366-426e3077-c65e-4ccb-a2ee-eeda6d1f1365.jpg)

And when you have more people inside the room I can't see them
Expected behavior

Still see avatars in the listing, validate the attributions field on import/update

Hardware

Device: Desktop
OS: Windows
Browser: Firefox

**Comment by kursr** [comment]
Yeah. We had to delist avatars with broken attributions as well to prevent the whole avatar-listing from exploding.
